wled00/bus_manager.cpp

```diff
@@ -401,14 +402,23 @@ BusPwm::BusPwm(BusConfig &bc)
 {
   if (!isPWM(bc.type)) return;
   unsigned numPins = numPWMPins(bc.type);
+#ifdef ESP8266
```

Comment:
Perhaps it would be best to completely split 8266 and 32 processing.

Reply:
agreed.